graphics/pixelformat.h

```diff
@@ -111,7 +120,7 @@ struct ColorComponent<7> {
 		       (value >> 6);
 	}
 };
-
+/** Return.... @todo not sure what that does if it's already 8-bit? */
```

Comment:
It's just returning the given value (and making it doesn't use more than 8 bits).
I think it is mostly here so that 8 bits component do not have to be treated in a specific way and we can use the same code for all the cases.